server/src/main/java/keywhiz/commands/DbSeedCommand.java

```diff
@@ -104,6 +104,11 @@ private static String computeRowHmac(String table, long id1, long id2) {
     return computeHmac(hmacContent.getBytes(UTF_8));
   }
 
+  private static String computeRowHmac(String table, String content, String metadata, long id) {
```

Comment:
I still think `computeRowHmac(String table, Vector<String> fields)` would be cleaner.